core/src/test/java/org/hibernate/ogm/backendtck/queries/AggregateOperationQueryTest.java

```diff
@@ -52,6 +52,30 @@ public void shouldCountEntities() {
 		} );
 	}
 
+	@Test
+	public void shouldCountDistinct() {
+		inTransaction( em -> {
+			Number result = (Number) em.createQuery( "SELECT COUNT(author.name) FROM Author author" ).getSingleResult();
```

Comment:
Just a few quick questions, I haven't checked the code in detail:

- Shouldn't the tests look like ```SELECT COUNT( DISTINCT author.name ) ...```?

- I would also expect to see a couple of tests running the count on the same column with `COUNT` and `COUNT(DISTINCT  ...)` showing different results.

Am I missing anything?

Reply:
Oh sorry, I forget about 'distinct' word. There are some changes should be done